src/main/index.js

```diff
@@ -405,11 +405,23 @@ function runApp() {
     event.returnValue = app.getPath('userData')
   })
 
+  ipcMain.handle(IpcChannels.GET_PICTURES_PATH, () => {
+    return app.getPath('pictures')
+  })
+
   ipcMain.handle(IpcChannels.SHOW_OPEN_DIALOG, async (_, options) => {
     return await dialog.showOpenDialog(options)
   })
 
-  ipcMain.handle(IpcChannels.SHOW_SAVE_DIALOG, async (_, options) => {
+  ipcMain.handle(IpcChannels.SHOW_SAVE_DIALOG, async (event, { options, modal }) => {
```

Comment:
For a boolean variable `modal` is not a good name for me
I wonder why modal is passed into the event in the first place o_0


```suggestion
  ipcMain.handle(IpcChannels.SHOW_SAVE_DIALOG, async (event, { options, useModal }) => {
```